presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -263,6 +264,14 @@ public void checkCanRenameColumn(ConnectorSecurityContext context, SchemaTableNa
         }
     }
 
+    @Override
+    public void checkCanSetTableAuthorization(ConnectorSecurityContext context, SchemaTableName tableName, PrestoPrincipal principal)
```

Comment:
Please add a product tests for that (as we don't have unit tests for sql-standard yet).